README.md

```diff
@@ -27,17 +27,17 @@ Ruby: 1.9.3, 2.X, Rails: >= 4.0. Works in pair with **Devise**, **Authlogic**, *
 
 ## Installation
 
- 1. List  Add to your Gemfile:
+ 1. List  Add to your Gemfile to the `:development` group:
 
-        gem 'any_login'
+        gem 'any_login', group: :development
 
  2. Execute in console:
 
         bundle install
 
  3. In application layout (for example `app/views/layouts/application.html.erb`) add in the bottom of the page:
 
-        <%= any_login_here %>
+        <%= any_login_here if Rails.env.development? %>
```

Comment:
but then we are loosing meaning of "enabled" option, because I know people (including me) may be using this gem for production. At least temporary. I like how you did try_not_to_leak_any_login_is_installed. Maybe it's enough?